scripts/2015_nips_paper/run/score_ensemble.py

```diff
@@ -102,9 +102,11 @@ def main(input_directories, output_file, task_id, seed, ensemble_size, n_jobs=1)
     losses = []
     top_models_at_step = dict()
 
-    backend = create(input_directory, input_directory + "_output",
-                     delete_tmp_folder_after_terminate=False,
-                     shared_mode=True)
+    backend = create(
```

Comment:
Sorry, might be a stupid question, but don't we require a prefix here, too?

Reply:
We do, this script is broken I'm fairly sure but I'll fix this